_pages/channels/truebeginners.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: channel
+title: TrueBeginners
+channel: truebeginners
+permalink: /channels/truebeginners/
+parent: channels
+---
+
+A "No Judgement Zone" for newbies, a good place to get your feet wet and ask all kinds of questions as to where to start, or for clarification on things that may be confusing.
```

Comment:
Can we add something about how we’re _all_ newbies in one topic or another? I think that does two things:
1. makes _true_ beginners feel like they’re not alone (“hey, even these experienced devs consider themselves newbs in _some_ capacity”)
2. makes experienced devs feel welcome to loosen up and ask questions